Week2/part2/app.js

```diff
@@ -0,0 +1,46 @@
+var mysql = require('mysql');
+var connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: '123123',
+  database: 'new_world',
+});
+
+connection.query(
+  `SELECT countrylanguage.CountryCode AS code, country.Name, COUNT(countrylanguage.language) AS total FROM countrylanguage JOIN country ON countrylanguage.CountryCode = country.Code GROUP BY (countrylanguage.CountryCode) `,
```

Comment:
How would you achieve this using triggers?
https://github.com/HackYourFuture/databases/tree/master/Week2#triggers